src/commands/subcommands/sections.ts

```diff
@@ -0,0 +1,81 @@
+import { HttpStatusCode, IModify } from '@rocket.chat/apps-engine/definition/accessors';
+import { SlashCommandContext } from '@rocket.chat/apps-engine/definition/slashcommands';
+import { LayoutBlock } from '@rocket.chat/ui-kit';
+
+import { TodoistApp } from '../../../TodoistApp';
+import { MiscEnum } from '../../enums/Misc';
+import {
+  getActionsBlock,
+  getButton,
+  getContextBlock,
+  getSectionBlock,
+} from '../../helpers/blockBuilder';
+import { getSectionsUrl } from '../../helpers/const';
+import { ISection } from '../../interfaces/sections';
+
+export async function sections(
```

Comment:
Take a look at what I said about labels, I think it is going to be usefull here as well :)